src/Command/CloudAccount/Backup/Download.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * @package Nexcess-CLI
+ * @license https://opensource.org/licenses/MIT
+ * @copyright 2018 Nexcess.net, LLC
+ */
+
+declare(strict_types = 1);
+
+namespace Nexcess\Sdk\Cli\Command\CloudAccount\Backup;
+
+use Closure;
+
+use Nexcess\Sdk\ {
+  Resource\CloudAccount\Backup,
+  Resource\CloudAccount\Endpoint,
+  Resource\Promise,
+  Util\Config,
+  Util\Util
+};
+
+use Nexcess\Sdk\Cli\ {
+  Command\CloudAccount\CloudAccountException,
+  Command\Create as CreateCommand,
+  Console
+};
+
+use Symfony\Component\Console\ {
+  Input\InputArgument as Arg,
+  Input\InputInterface as Input,
+  Input\InputOption as Opt,
+  Output\OutputInterface as Output
+};
+
+/**
+ * Creates a new Cloud Account.
+ */
+class Download extends CreateCommand {
+
+  /** {@inheritDoc} */
+  const ARGS = [];
+
+  /** {@inheritDoc} */
+  const ENDPOINT = Endpoint::class;
+
+  /** {@inheritDoc} */
+  const SUMMARY_KEYS = ['filename'];
+
+  /** {@inheritDoc} */
+  const INPUTS = [];
+
+  /** {@inheritDoc} */
+  const NAME = 'cloud-account:backup:download';
+
+  /** {@inheritDoc} */
+  const OPTS = [
+    'cloud_account_id|c' => [OPT::VALUE_REQUIRED],
```

Comment:
for consistency, i'd like all cli options to have the same format. existing, as well as built-in symfony options, use `--dashed-option-names`.